src/components/ui/Checkbox.module.scss

```
@@ -0,0 +1,49 @@
+  border-radius: 6px;
+}
+
+.container {
+  position: relative;
+  display: flex;
+  gap: 10px;
+  align-items: center;
+  user-select: none;
+  cursor: pointer;
+
+  .checkbox :global(.icon) {
```

Stylelint (17.3.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)